.project-template/fill_template_vars.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import re
+from pathlib import Path
+
+
+def _find_files(project_root):
+    exclude_pattern = r"\.git($|\/)|venv"
```

Comment:
This may be a special case for me, but I'm getting `UnicodeDecodeErrors` on the `docs/_build` files, and my vim `.swp` files that are open. I think if you add those paths to the `exclude_pattern` you should be good to go!

Reply:
Added!